specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/virtualMachineScaleSet.json

```diff
@@ -3651,13 +3651,16 @@
           "type": "integer",
           "format": "int32",
           "minimum": 0,
+          "maximum": 1000,
```

Comment:
Let's not drive-by add this. As I said elsewhere, I don't think this needs to be hardcoded given that the limit is somewhat dynamic and in process of being lifted (the situation is somewhat complex). Plus, I don't think `sku.capacity` has this in the API spec, so we don't need it here either (they're the same limit). Let's keep this change scoped to fixing PATCH/update.